### PR TITLE
LPS-114059 - Apply the use of sheet clay taglib in staging-taglib

### PR DESCRIPTION
**Pull request by marcoscv-work**

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:stable

**Comment by marcoscv-work** [comment]
ci:test:relevant

File: modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/review_uad_data.jsp [hunk 1]
```diff
@@ -185,20 +185,22 @@ renderResponse.setTitle(StringBundler.concat(selectedUser.getFullName(), " - ",
 		<clay:col
 			lg="9"
 		>
-			<div class="sheet">
-				<div class="sheet-header">
+			<clay:sheet
+				size="<%= null %>"
```
Comment (marcoscv-work) [inline modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/review_uad_data.jsp:189]: I cannot see this JSP:
http://localhost:8080/group/control_panel/manage?p_p_id=com_liferay_user_associated_data_web_portlet_UserAssociatedData

the compilation works fine, @jbalsas could you tell me if this is correct?

**Comment by jbalsas** [comment]
Screen Shot 2020-05-25 at 18 09 42

Please fix this and send again 😉

**Pull request closed without merging**